### ISSUE TITLE
Should work on node 0.10

### ISSUE THREAD

**Issue by mmalecki** [nodejitsu/godot#55]
Anyone wants this issue? Seems fairly easy, but tests still fail.

**Comment by edef1c** [comment]
This seems to be all that fails

```
  ♢ godot/net/client-reconnect 

  Godot client with no backoff and no server
    ✓ should emit an error
    ✓ should emit an error
✗ Errored » callback not fired 
      in Godot client with backoff and no server 
      in godot/net/client-reconnect 
      in test/net/client-reconnect-test.js
✗ Errored » callback not fired 
      in Godot client with backoff and server eventually coming up 
      in godot/net/client-reconnect 
      in test/net/client-reconnect-test.js
```

**Comment by jcrugzz** [comment]
@nathan7 can you figure out if thats localized in `node-backoff` or just `vows` failing on node 0.10?

**Comment by jcrugzz** [comment]
Only tests that fail on the [`0.10.x`](https://github.com/nodejitsu/godot/tree/0.10.x) branch are the above tests which have a high probability of being caused by vows.

**Comment by jcrugzz** [comment]
This should be fixed once I close #58
